### PR TITLE
Fix for android include pthread.h from /usr/include

### PR DESCRIPTION
**Pull request by gelldur**
Fix forr issue #1350

Sorry if I pr to wrong branch. ( I can fix that if everything is ok )

### CODE REVIEW

**Comment by gelldur** [comment]
Any comments?

**Comment by obiltschnig** [comment]
From my point of view this looks ok, but maybe @bachp has some comments? Also, please re-submit for develop branch.

File: CMakeLists.txt [hunk 1]
```diff
@@ -167,18 +167,20 @@ if(WIN32 AND ENABLE_NETSSL_WIN)
...
-    add_subdirectory(NetSSL_OpenSSL)
-    list(APPEND Poco_COMPONENTS "NetSSL_OpenSSL")
-    endif()
-    if(ENABLE_CRYPTO)
-    add_subdirectory(Crypto)
-    list(APPEND Poco_COMPONENTS "Crypto")
-    endif()
-endif(OPENSSL_FOUND)
+if(ENABLE_NETSSL OR ENABLE_CRYPTO)
+    find_package(OpenSSL)
```
Comment (bachp) [inline CMakeLists.txt]: I would mark this as required. So it fails. Otherwise it is confusing as it fails silently.

**Comment by gelldur** [comment]
Fixed according to code review and changed base of PR to `develop`

**Pull request merged**